src/utils/Either.ts

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
Please create a story to update fp-ts to v2 and remove this file.

Reply:
1.17.x is the first version where we have these functions (in case 2.x differs too much):

https://github.com/gcanti/fp-ts/blob/1.17.0/src/Either.ts

Reply:
I can update to v1.17 into this PR and remove the file @francescopersico

Reply:
Fixed in e4d7db4

Reply:
Now the version of `fp-ts` for `italia-ts-commons` and `io-backend` are not aligned.
I've created a story to fix this [here](https://www.pivotaltracker.com/story/show/169728543)